charts/alfresco-transform-service/values.yaml

```
@@ -130,6 +130,43 @@ pdfrenderer:
     maxTransformSeconds: 1200
   volumes: []
   volumeMounts: []
+  autoscaling:
+    # -- Toggle pdfrenderer autoscaling
+    enabled: false
+    # -- minimum number of replicas to spin up within the replicatset
+    minReplicas: 1
+    # -- maximum number of replicas to spin up within the replicatset
```

```suggestion
    # -- minimum number of replicas to spin up within the replicaset
    minReplicas: 1
    # -- maximum number of replicas to spin up within the replicaset
```

fix also in the others sections :D